pages/design/rule-design.md

```diff
@@ -39,34 +63,56 @@ A test aspect is a part of a test subject that must be available in order to pro
 
 While most assumptions relate to the rule itself, there are some assumptions that apply at other stages of the evaluation:
 
 - It is assumed that the tested web page is the one that has to conform to WCAG 2.0 and that there is no [conforming alternative version](https://www.w3.org/TR/WCAG21/#dfn-conforming-alternate-version).
 
 - It is assumed that the following technologies are accessibility supported: HTML, CSS, WAI-ARIA, ... (See also WCAG-ACT-RULES-CG's [explanation on Accessibility Support](accessibility-support.html)).
 
-## Applicability
+For more details, see [ACT Rules Format: Assumptions](https://www.w3.org/TR/act-rules-format/#assumptions)
 
-Applicability describes which (elements of) web pages should be tested using the rule. These elements are known as test targets. Applicability must be written in plain language, so that it can be used by QA testers. Applicability must rely on well defined properties of the technologies that are tested. For example, a rule may be applicable to all `video` elements, but it can not be applicable to all `object` elements used to show video, unless the term "video" is further defined.
+## Accessibility Support
 
-Finding objective definitions to use in rules can be difficult, if not outright impossible in some cases. The intent here is to ensure repeatability of the rule. Not everything in WCAG testing is entirely repeatable, but when it comes to rule applicability, this is a hard requirement.
+Information on any known support issues for a feature relevant to the rule.
 
-For more details, see [ACT Rules Format: Applicability](https://w3c.github.io/wcag-act/act-rules-format.html#applicability)
+For more details, see [ACT Rules Format: Accessibility Support](https://www.w3.org/TR/act-rules-format/#accessibility-support)
 
-## Expectations
+## Background
 
-The applicability help the testers (or test tools) identify what has to be checked. Following that, the expectations are statements that must be true for the applicable elements to pass the rule. If any of the expectations is false, than the target element failed the rule.
+Optional links to information that is relevant reading, such as a success criterion, understanding or technique document, or other specification. 
 
-Each expectation exposes a reason why an element may not meet a particular conformance requirement. The expectations can be "linked", in that one has to be met before a second can be tested. For example, a rule testing link names may have as its first expectation "Target element has an accessible name", and as a second expectation "Expectation 1 is true for the target element, and the accessible name describes the function of the target element".
+For more details, see [ACT Rules Format: Background](https://www.w3.org/TR/act-rules-format/#background)
+
+## Test cases
```

Comment:
Add something about example descriptions. First that they must be here and explain a bit what happen. More ideas on #834 but that's also something that need more work and agreeing first…

Reply:
Just checking... this is in reference to the line of text between Passed/Failed/Inapplicable Example _n_ and the actual code snippet?

Reply:
Yes.